src/preloadjs/LoadQueue.js

```diff
@@ -1222,14 +1222,14 @@ this.createjs = this.createjs || {};
 	 * @method getItems
 	 * @param {Boolean} loaded Determines if only items that have been loaded should be returned. If false, in-progress
 	 * and failed load items will also be included.
-	 * @returns {Array} A list of objects that have been loaded. Each item includes the {{#crossLink "LoadItem"}}{{/crossLink}},
+	 * @returns {Object[]} A list of objects that have been loaded. Each item includes the {{#crossLink "LoadItem"}}{{/crossLink}},
```

Comment:
While YUIDocs compiles and displays this as "Object[]", I don't think it is the best approach, since JavaScript doesn't support typed arrays, and it will link to this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object

I recommend removing this usage.